user/how-to-guides/how-to-use-usb-devices.md

```diff
@@ -28,9 +28,9 @@ Examples of valid cases for USB-passthrough:
 (If you are thinking to use a two-factor-authentication device, [there is an app for that](/doc/u2f-proxy/).
 But it has some [issues](https://github.com/QubesOS/qubes-issues/issues/4661).)
 
-## Attaching And Detaching a USB Device
+## Attaching and detaching a USB device
 
-### With Qubes Device Manager
+### With Qubes device manager
```

Comment:
I'm not certain, but I think "Qubes Device Manager" is the name of the thing, so should remain in title case.

Reply:
Yes, you are right. That was a mistake.